install.sh

```diff
@@ -126,17 +126,25 @@ __install() {
 			printf "$_ITEM Bro IDS is already installed \n"
 		fi
 
+		if [ "$_BRO_INSTALLED" = "true" ]; then
```

Comment:
Previous installs will already have bro. We still want the start on boot settings to be applied to them.

Accidentally deleted @ethack's comment. Replaced here.